config-connector/solutions/sql/helm/postgres-ha/README.md

```diff
@@ -0,0 +1,93 @@
+# PostgreSQL High Availability
+
+==================================================
+
+## NAME
+
+  postgres-ha
+
+## SYNOPSIS
+
+  Config Connector compatible yaml files to configure a high availability PostgreSQL cluster.
+
+## CONSUMPTION
+
+  1. Clone GoogleCloudPlatform/cloud-foundation-toolkit repository:
+
+      ```bash
+      git clone https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit.git
+      ```
+
+  1. Go to the service account folder:
+
+      ```bash
+      cd cloud-foundation-toolkit/config-connector/solutions/sql/helm/postgres-ha
+      ```
+
+## REQUIREMENTS
+
+1. GKE Cluster with Config Connector and [Workload Identity](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity#enable_workload_identity_on_a_new_cluster).
+1. [Helm](../../../README.md#helm)
+1. A working Config Connector instance.
```

Comment:
I think this is duplicate with the first numbered item. Maybe you can remove this one and reword the first numbered item a bit. For example, `GKE Cluster with [Config Connector installed using a GKE Workload Identity](https://cloud.google.com/config-connector/docs/how-to/install-upgrade-uninstall#workload-identity).`